stl/inc/format

```diff
@@ -507,7 +522,11 @@ protected:
 
     _Fmt_codec_base() {
 #ifndef _FORMAT_CODEPAGE

```

Comment:
I believe we should now be able to drop the "override" logic in:

https://github.com/microsoft/STL/blob/fcd5cf555f073fe0bfc49a170ce95aa784f6e704/tests/std/tests/P0645R10_text_formatting_legacy_text_encoding/test.cpp#L4